.svglintrc.mjs

```diff
@@ -1,11 +1,26 @@
-const fs = require('fs');
-
-const data = require('./_data/simple-icons.json');
-const { htmlFriendlyToTitle } = require('./scripts/utils.js');
-const htmlNamedEntities = require('named-html-entities-json');
-const svgpath = require('svgpath');
-const svgPathBbox = require('svg-path-bbox');
-const parsePath = require('svg-path-segments');
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
```

Comment:
Since we required Node.js `^12.20.0 || ^14.13.1 || >= 16.0.0` for running this script. We can use [`node:` protocol](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-move-my-commonjs-project-to-esm) for imports.

```js
import fs from 'node:fs';
import path from 'node:path';
import { fileURLToPath } from 'node:url';
```

Here is a [discussion](https://github.com/nodejs/node/issues/38343) about `node:` procotol.

Reply:
Oh, thanks for looking into that, I didn't use them because I wasn't sure if they worked on all of our supported versions. I'll change all of the std lib imports to use the `node:` prefix.